src/extension/analytics.js

```diff
@@ -16,7 +16,7 @@ async function getAnalyticsVisitor() {
   const visitor = analytics(analyticsTrackingId, clientId);
   reportData.clientId = clientId;
 
-  const email = await getLocalDataClient().loadUserEmail();
+  const { email } = await getValue('developer') || {};
```

Comment:
be consistent. You are using both cache.getValue and getValue. `cache.getValue` look better